nassima-nassBouz/week-2/js-racer-game-lab/starter-code/app.js

```diff
@@ -0,0 +1,18 @@
+let evRed = document.querySelector('.start');
+evRed.addEventListener('click', moveRed);
+
+function moveRed() {
+  var elem = document.getElementById("myAnimation");
```

Comment:
Just a little tip when you have a file using ES6 or ES5 syntax it is standard to use the same syntax throughout the file. So only use let or only use var. This habit will save you a lot of headaches due to the difference between var and let. :octocat: